src/codecoverage/backend/tests/test_coverage.py

```diff
@@ -28,6 +31,79 @@ def test_coverage_supported_extensions_api(client):
     assert data['previous_rev'] == '5401938bde37d0e7f1016bbd7694e72bdbf5e9a1'
 
 
+def mock_coverage_by_changeset_job_success(job_changeset):
+    from tests.conftest import coverage_builds as get_coverage_builds
```

Comment:
@marco-c because of this line, it failed in taskcluster if I didn't add __init__.py
I'll try to remove it and restart the test

Reply:
even though it success in my local without the init.py :/

Reply:
OK, let's keep it then